volontulo/templates/offers/gallery.html

```diff
@@ -0,0 +1,30 @@
+{% load bootstrap3 %}
+
+<div class="form-group form-group-sm">
+    <label class="col-xs-2 control-label" for="{{ offer_form.benefits.id_for_label }}">Galeria oferty</label>
+    <div class="col-xs-4">
+        <form action="" method="post" enctype="multipart/form-data" role="form">
+            {% csrf_token %}
+            {% bootstrap_form offer_image_form %}
+            <p class="form-buttons">
```

Comment:
`<p>` is for text paragraphs. If a form element has to be wrapped to generate space, it could be a `div` width a class `form-group`